notebook/static/base/js/keyboard.js

```diff
@@ -225,6 +225,17 @@ define([
         }
         return dct;
     };
+    
+    ShortcutManager.prototype.get_shortcut_for_action_name = function(name){
```

Comment:
I guess the in this file we are using underscore separated words, so `get_shortcut` ... :/